### ISSUE TITLE
StringIndexOutOfBoundsException in PrimaryProcessor

### ISSUE THREAD

**Issue by g4s8** [artipie/rpm-adapter#57]
The file `v8location_rootfs-1213-lpu-1.0.0-3.aarch64.rpm` causes unexpected failure in RPM adapter (version `0.51.1`), exception:
```
'String index out of range: 0'
java.lang.StringIndexOutOfBoundsException: String index out of range: 0
        at java.lang.String.charAt(String.java:658) ~[?:1.8.0_212]
        at com.artipie.rpm.PrimaryProcessor.writeFiles(PrimaryProcessor.java:298) ~[rpm-adapter-0.51.1.jar:?]
        at com.artipie.rpm.PrimaryProcessor.lambda$null$5(PrimaryProcessor.java:199) ~[rpm-adapter-0.51.1.jar:?]
        at io.reactivex.internal.operators.completable.CompletableFromAction.subscribeActual(CompletableFromAction.java:34)
```

**Comment by 0crat** [comment]
The job [#57](https://github.com/artipie/rpm-adapter/issues/57) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/e0782451-7fd4-4f9b-80c6-b65051c5ab57); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by g4s8** [comment]
@rultor release, tag=`0.5.2`

**Comment by g4s8** [comment]
@rultor release, tag=`0.5.3`

**Comment by g4s8** [comment]
Fixed in `0.5.3`

**Comment by 0crat** [comment]
Job was finished in 31 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))
